From c3be8e02a600e8e9097f61bed07b398dc4e2d641 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 3 Jun 2018 18:34:25 +0200
Subject: [PATCH 1/1] scripts: mailmapper: SPDX license identifier

If the SPDX license identifier is in the first line the shell does not
recognize which interpreter shall be used to execute the script.

Cf. https://www.kernel.org/doc/html/v4.16/process/license-rules.html
for scripts which require the '#!PATH_TO_INTERPRETER' in the first line
(...) the SPDX identifier goes into the second line.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 scripts/mailmapper | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/mailmapper b/scripts/mailmapper
index f73f3dc1787..2e2d7faff57 100755
--- a/scripts/mailmapper
+++ b/scripts/mailmapper
@@ -1,5 +1,5 @@
-# SPDX-License-Identifier: GPL-2.0+
 #!/usr/bin/env python2
+# SPDX-License-Identifier: GPL-2.0+
 #
 # Copyright (C) 2014, Masahiro Yamada <yamada.m@jp.panasonic.com>
 
-- 
2.17.1

